### PR TITLE
Event Modelling

### PR DESCRIPTION
**Pull request by davidoj**
I have left out a lot of stuff for now as I'm not sure how to display it effectively. 

Expected numbers of infections are in most cases very low (small chance for someone to be at the event + small chance of transmission)

### CODE REVIEW

**Comment by hnykda** [comment]
Should this be merged? Is it ready

**Comment by Bachibouzouk** [comment]
Could you summarize conceptually what you did in the CHANGELOG.md file (following the format of previous lines) ? Is this PR linked to an issue?

File: src/server/templates/result-event-evaluation.html [hunk 1]
```diff
@@ -3,72 +3,65 @@
...
+  <!-- TODO: explain calculations -->
 
+
+  <!-- TODO: this outputs 0.0 when there is an error -->
   <label>
-    This means your event will probably infect
-    <span class="min-infect"></span>
-    to
-    <span class="max-infect"></span>
+    Under the conditions of your country, saving one person from being infected at your event means
```
Comment (lagerros) [inline src/server/templates/result-event-evaluation.html]:
```suggestion
    In {{ place }}, saving one person from being infected at your event means
```

File: src/server/templates/result-event-evaluation.html [hunk 2]
```diff
@@ -3,72 +3,65 @@
...
-  <button>See calculation v</button>
 
+  {% if excess_hospital_load != 'Not applicable' %}
   <label>
-    They will probably go on to infect
-    <span class="min-infect-2nd-order"></span>
-    to
-    <span class="max-infect-2nd-order"></span>
-    people
+    Under the conditions of your country, saving one person from being infected at your event means
```
Comment (lagerros) [inline src/server/templates/result-event-evaluation.html]:
```suggestion
    In {{ place }}, saving one person from being infected at your event means
```

**Comment by davidoj** [comment]
Connected to this issue, but I don't have rights to link them https://github.com/epidemics/covid/issues/11

I haven't changed CHANGELOG.md as it didn't seem to be a place for conceptual descriptions. I have added comments to the issue.

**Comment by davidoj** [comment]
I've added gleamviz data to the event prediction, but I've introduced a bug whereby the selection options are ignored. I can't work out how I've done it tonight.

**Comment by hnykda** [comment]
I rebased on master here: https://github.com/epidemics/covid/pull/134 . So closing this one and let's continue there.

I also added you as a collaborator, so you should be able to clone this repo and continue on that branch instead in your own fork.
